plugin/online-thesaurus.vim

```diff
@@ -61,6 +62,21 @@ function! s:Lookup(word)
     exec 'resize ' . (line('$') - 1)
     setlocal nomodifiable filetype=thesaurus
     nnoremap <silent> <buffer> q :q<CR>
+    let @@ = reg_save
+endfunction
+
+function! s:LookupVisual(mode)
+    " Only support character-wise visual mode
+    if a:mode == 'v'
+        let reg_save = @@
```

Comment:
This still affects register `"*` in my setup. I guess it's better to use a named register like `@a`